azure-devops/azext_devops/dev/team/wiki.py

```diff
@@ -0,0 +1,220 @@
+    organization, project = resolve_instance_and_project(detect=detect,
+                                                         organization=organization,
+                                                         project=project)
+    wiki_client = get_wiki_client(organization)
+    from azext_devops.devops_sdk.v5_0.wiki.models import WikiPageCreateOrUpdateParameters
+    parameters = WikiPageCreateOrUpdateParameters()
+    if content:
+        parameters.content = content
+    if file_path:
+        fp = open(file_path, mode='r')
+        parameters.content = fp.read()
+        fp.close()
+    return wiki_client.create_or_update_page(parameters=parameters, wiki_identifier=wiki,
+                                             project=project, path=path, version=None, comment=comment)
+
+
+def update_page(wiki, path, comment=_DEFAULT_PAGE_UPDATE_MESSAGE, content=None, file_path=None,
+                page_version=None, organization=None, project=None, detect=None):
+    """Edit a page.
+     :param wiki: Name or Id of the wiki.
+    :type wiki: str
+    :param path: Path of the wiki page.
+    :type path: str
+    :param content: Content of the wiki page. Ignored if --file-path is specified.
+    :type content: str
+    :param file_path: Path of the file input if content is specified in the file.
+    :type file_path: str
+    :param comment: Comment in the commit message of file edit operation.
+    :type comment: str
+    :param page_version: Version of file to edit.
+    :type page_version: str
+    """
+    if not content and not file_path:
+        raise CLIError('Either --file-path or --content must be specified.')
+    organization, project = resolve_instance_and_project(detect=detect,
+                                                         organization=organization,
+                                                         project=project)
+    wiki_client = get_wiki_client(organization)
+    from azext_devops.devops_sdk.v5_0.wiki.models import WikiPageCreateOrUpdateParameters
+    parameters = WikiPageCreateOrUpdateParameters()
+    if content:
+        parameters.content = content
+    if file_path:
+        fp = open(file_path, mode='r')
+        parameters.content = fp.read()
+        fp.close()
+    return wiki_client.create_or_update_page(parameters=parameters, wiki_identifier=wiki,
+                                             project=project, path=path, version=page_version, comment=comment)
+
+
+def get_page(wiki, path, version=None, recursion_level=None, open=False,  # pylint: disable=redefined-builtin
+             include_content=False, organization=None, project=None, detect=None):
+    """Get the content of a page or open a page.
+    :param wiki: Name or Id of the wiki.
+    :type wiki: str
+    :param path: Path of the wiki page.
+    :type path: str
+    :param version: Version of the wiki page.
+    :type version: str
+    :param recursion_level: Recursion level of the wiki page.
```

Comment:
a bit confused, this is get_page or get_pages 🤣  (due to recursion_level)

Reply:
also I believe recursion_level has some fixed set of values allowed (please check once) and we can make this an enum

Reply:
API is same. This is just the wrapped up API so let us discuss this in bash and close who we want the get pages to be exposed.